package.json

```
@@ -68,6 +68,7 @@
     "graasp-plugin-actions": "github:graasp/graasp-plugin-actions",
     "graasp-plugin-categories": "github:graasp/graasp-plugin-categories",
     "graasp-plugin-chatbox": "github:graasp/graasp-plugin-chatbox",
+    "graasp-plugin-etherpad": "github:graasp/graasp-plugin-etherpad#1-add-etherpad",
```

- TODO: once https://github.com/graasp/graasp-plugin-etherpad/pull/3 is merged
```suggestion
    "graasp-plugin-etherpad": "github:graasp/graasp-plugin-etherpad#1-add-etherpad",
```